src/LayerEditor/ui/diagram_editor/graphics_items/gs_point.py

```diff
@@ -15,6 +16,7 @@ class GsPoint(QtWidgets.QGraphicsEllipseItem):
     no_brush = QtGui.QBrush(QtCore.Qt.NoBrush)
     no_pen = QtGui.QPen(QtCore.Qt.NoPen)
     add_brush = QtGui.QBrush(QtCore.Qt.darkGreen, QtCore.Qt.SolidPattern)
+    dim = 0
```

Comment:
??

Reply:
It is very convenient for working with GS objects, without having to find out whether it is Point, Segment, or Polygon